### ISSUE TITLE
Confusing name for global configuration: "renderProfile"

### ISSUE THREAD

**Issue by eordano** [decentraland/unity-renderer#133]
Due to the lack of context of `explorer.json`; the field "renderProfile" doesn't communicate well what it does.

- It's a 0. Could this mean a boolean value? Is this a disabled feature?
- Does the "profile" there mean that this disables rendering of other user's Profiles?
- After taking a look at what it does, it sets night mode as a default when it's set to 1.

I think this is going to be super tricky to understand in the future. After taking a look at what it does, it's an "aha! it's the default _rendering_ profile to be used!". Let's fix this for more understandable configuration:

- Change the name from "renderProfile" to "defaultDayNightMode", or "defaultEnvironmentalRenderingSetup" (avoiding Profile to prevent confusion with avatars or other people, avoiding Settings because of all the settings the user can select) for the name of the configuration, if we're looking to have more rendering profiles in the future.
- Change the value from 0 or 1 to "day" and "night" (make it an enumeration in the config, not in the code)

TL;DR: Pay tech debt, turn this
```
{
  "renderProfile": 1
}
```
Into this:
```
{
  "defaultDayNightMode": "night"
}
```
So it's more understandable for someone modifying the client

**Comment by JorgeDecentraland** [comment]
Obsolete due to procedural skybox
